scripts/build/termux_step_get_dependencies_bionic.sh

```diff
@@ -0,0 +1,137 @@
+__log() {
+	local msg="$1"
+	[[ "$TERMUX_QUIET_BUILD" != "true" ]] && echo "$msg"
+}
```

Comment:
Seems like it could be moved to https://github.com/termux/termux-packages/blob/master/scripts/utils/termux/package/termux_package.sh